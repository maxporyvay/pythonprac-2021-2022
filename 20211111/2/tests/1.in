a-w0. -q[35iq[5/iq33v05p mjt,p9
(0,0), (0,1), (1,0)
