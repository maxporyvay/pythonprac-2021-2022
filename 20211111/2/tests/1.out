Invalid input
0.50
